products/chef-infra-server.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Chef Infra Server
+category: server-app
+iconSlug: chef
+tags: erlang-runtime ruby-runtime
+permalink: /chefinfraserver
+alternate_urls:
+-   /chefserver
+releasePolicyLink: "https://docs.chef.io/versions/"
+changelogTemplate: "https://docs.chef.io/release_notes_server/#__LATEST__/"
+versionCommand: chef-server-ctl version
+releaseDateColumn: true
+activeSupportColumn: true
+
+auto:
+-   git: https://github.com/chef/chef-server
```

Comment:
The repository doesn't represent the actual versions. Each PR that is merged automatically bumps the patch release, minor release, or major release depending on GitHub labels that are applied. When the team is ready they promote these versions so you end up with versions like 1.0.15 and 1.1.24 being the released versions. There is an API called omnitruck that will tell you stable release versions.

https://docs.chef.io/api_omnitruck/

Reply:
Thank you for your feedback, I didn't have this information. I [quickly looked](https://omnitruck.chef.io/stable/chef/metadata?p=debian&pv=11&m=x86_64) at Omnibus, I'm afraid that the latest versions differ depending on the OS & platform

I think it would be best and the simplest to get the latest version from the [product release page](https://docs.chef.io/release_notes_server/#15.9.20), WDYT?

I have the impression that there is a release note for each version.   
Based on this, we could use this script to update the data.

```python
import requests
from bs4 import BeautifulSoup

url = "https://docs.chef.io/release_notes_server/"
response = requests.get(url)

if response.status_code == 200:
    soup = BeautifulSoup(response.text, 'html.parser')
    h2_elements = soup.find_all('h2', text=lambda text: "Chef Infra Server" in text)
    for h2_element in h2_elements:
        version_id = h2_element.get('id')
        print(f"{version_id}")
else:
    print(f"{response.status_code}")
```

In both cases it is possible to retrieve the release date of the latest versions, so I removed the field.

Reply:
Unfortunately the date is not documented on https://docs.chef.io/release_notes_server/, so you can't use this alone to add some automation. Maybe by combining both this and information from https://github.com/chef/chef-server/tags ?

Reply:
Good idea, I hadn't thought of that, I did that and it works, I'm going to open a PR on the other repo